backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/entities/Token.java

```diff
@@ -0,0 +1,33 @@
+package com.google.gpay.virtualqueue.backendservice.entities;
+
+public class Token {
+    private Integer shopId;
+    private Integer tokenNo;
+    private Boolean status;
```

Comment:
Can we have an enum for status, instead of having a boolean?
enum {
STATUS_UNKNOWN = 0,
ACTIVE = 1,
EXPIRED = 2,
CANCELLED_BY_CUSTOMER = 3,
CANCELLED_BY_SHOP_KEEPER = 4
}

something like this.
This is just a suggestion

Reply:
yeah that looks great! I'll add it!